README.md

```diff
@@ -101,6 +101,12 @@ with an existing Windows installation.
    iwr -useb 'https://github.com/rancher-sandbox/rancher-desktop/raw/main/scripts/windows-setup.ps1' | iex
    ```
 4. Close the privileged PowerShell prompt.
+5. Ensure `msbuild_path` and `msvs_version` are configured correctly in `.npmrc` file. Run below commands to set these properties.
```

Comment:
I would write this as:

Ensure `msbuild_path` and `msvs_version` are configured correctly in `.npmrc`. Run the following commands to set these properties: